util/domainutil/domain_vars.go

```diff
@@ -0,0 +1,178 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package domainutil
+
+import (
+	"strings"
+	"sync/atomic"
+
+	"github.com/pingcap/parser/model"
+)
+
+type repairInfo struct {
+	repairMode      atomic.Value
+	repairTableList atomic.Value
+	repairDBInfoMap atomic.Value
```

Comment:
It could lead to overwriting between threads.

Reply:
`admin repair table` job is executed sequentially, won't produce multi-write at a same time.